app/index.js

```diff
@@ -362,9 +362,31 @@ ipcMain.on('update-activities', () => {
   updateActivitiesWindow()
 })
 
+export const handleClearActivities = (activitiesById, activities) => {
```

Comment:
can we add more comments?

Reply:
We need a little more documentations on what this is doing for future works

Reply:
Yep, good idea :taco:

Reply:
Does this make more sense?

Reply:
Yes! :D